src/Controllers/PushController.js

```diff
@@ -84,6 +88,34 @@ export class PushController {
     }
     return expirationTime.valueOf();
   }
+
+  /**
+   * Get push time from the request body.
+   * @param {Object} request A request object
+   * @returns {Number|undefined} The push time if it exists in the request
+   */
+  static getPushTime(body = {}) {
+    var hasPushTime = !!body['push_time'];
```

Comment:
I've used the same convention found in the getExpirationTime function.

https://github.com/felipemobile/parse-server/blob/master/src/Controllers/PushController.js#L66